sdcm/cluster.py

```diff
@@ -1306,15 +1308,28 @@ def install_mgmt(self, scylla_repo, scylla_mgmt_repo):
         if not (self.is_rhel_like() or self.is_debian() or self.is_ubuntu()):
             raise ValueError('Unsupported Distribution type: {}'.format(str(self.distro)))
         if self.is_debian8():
+            self.remoter.run(cmd="sudo sed -i -e 's/jessie-updates/stable-updates/g' /etc/apt/sources.list")
+            self.remoter.run(
+                cmd="sudo echo 'deb http://archive.debian.org/debian jessie-backports main' | sudo tee -a /etc/apt/sources.list.d/jessie-backports.list")
+            self.remoter.run(cmd="sudo touch /etc/apt/apt.conf.d/99jessie-backports")
+            self.remoter.run(
+                cmd="sudo echo 'Acquire::Check-Valid-Until \"false\";' | sudo tee /etc/apt/apt.conf.d/99jessie-backports")
+            self.remoter.run('sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys F76221572C52609D', retry=3)
             install_transport_https = dedent("""
```

Comment:
why do you change dedent() back to multiple remoter run? it's retrogressing ;-)

Reply:
since Debian-8 is quite in an unstable support state and occasionally breaks compatibility, it is much clearer and helps identify failures not using dedent().

Reply:
I am not sure why it is more clear: the script is run in bash using `-x` that prints each command that it runs and `-e` that exits on first failure. Error is displayed last and each executed command is printed...